src/main/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltration.java

```diff
@@ -199,8 +198,7 @@ private String possiblyInvertFilterExpression( final String description ){
     }
 
     private void initializeVcfWriter() {
-        //TODO remove hardwiring to output VCFs
-        writer = new VariantContextWriterBuilder().setOutputFile(out).setOutputFileType(VariantContextWriterBuilder.OutputType.VCF).unsetOption(Options.INDEX_ON_THE_FLY).build();
+        writer = createVCFWriter(new File(out));
```

Comment:
Can you have a look at `HaplotypeCallerEngine.makeVCFWriter()` in gatk-protected, and see whether that method can be made to be consistent with your work in this branch, and create a new ticket to patch it? It calls `GATKVariantContextUtils.createVCFWriter()`, but sometimes it constructs a new `GVCFWriter`. You may need a way to create a `GVCFWriter` at the `GATKTool` level, as well as a static entry point not tied to the tool instance (but we can deal with this in a separate ticket)